### ISSUE TITLE
The last PR broke Ollama local model selection.

### ISSUE THREAD

**Issue by clementvp** [wandb/openui#116]
I've pull the newest version -> https://github.com/wandb/openui/commit/6f4a26dbadb5674ff825bb26f6f745fa57cd223f

I've resinstall the backend and set a Groq api key, but in the model selection dropwdown I just got OpenAi models.

No more Ollama locals models ( the previous version had that). ( Ollama running on my computer)
And no possibility to choose a groq model.

![image](https://github.com/wandb/openui/assets/11894024/9e4d44b8-b813-48aa-ae58-425d73f817c0)

**Comment by kakachiex2** [comment]
same problem here

**Comment by Abdulkader-Safi** [comment]
I there any way to solve it, Or not yet

**Comment by omkargadute** [comment]
I think its some problem in the UI, as the API is listing the groq models perfectly.

**Comment by clementvp** [comment]
> Is there any way to solve it, Or not yet

I have rollback to the previous commit ( but no grog support ).

**Comment by clementvp** [comment]
@tlo9 @vanpelt  Do you run on this issue too ? Are we missing something ? Thx again for the PR and the project  :)

**Comment by dbian** [comment]
You need to build the frontend manually for now

```shell
cd frontend
pnpm i
npm run deploy
npm run postbuild
```

Restart your container/app

**Comment by niktekusho** [comment]
Can confirm that @dbian solution works on my machine and I can finally use local models.

**Comment by clementvp** [comment]
I confirm that work 👍 thx @dbian

**Comment by DiazMichael** [comment]
Legend! @dblan
<details>
  <summary>Screenshot</summary>

![image](https://github.com/wandb/openui/assets/53813993/00a837a3-c18b-49bc-a068-4656c05eda8d)

</details>

**Comment by kakachiex2** [comment]
@dbian 
this process doesn't work for me I'm on windows 11

**Comment by kakachiex2** [comment]
the last part restart your container app you mean the "python -m openui"

**Comment by dbian** [comment]
> the last part restart your container app you mean the "python -m openui"

Yes, if you are running in container mode, you should use `docker-compose up -d --build` instead.

Feel free to cherry-pick my [commits ](https://github.com/dbian/openui/commit/f1702d83002a2e0e0d7c11a52af066ff1fee9164)if you encounter issues generating the frontend on Windows.

**Comment by vanpelt** [comment]
I am so sorry everyone.  I totally broke the `main` branch 😭 .  It's time I add some integration tests and prevent merges to master without them passing, this is amatuer hour.  Again I appologize.

I just pushed a number of changes and a fresh build of the UI to the main branch.  If you pull master and run the following from the backend dir, it should work without requiring a build of your own:

```bash
pip uninstall openui
pip install .
```

**Comment by clementvp** [comment]
@vanpelt don't be sorry, it's happen to everybody. Thank you for this awesome project ;)
